website/assets/script/code.js

```diff
@@ -1,25 +1,21 @@
-for (const elem of document.querySelectorAll('code.hljs')) {
-    const button = document.createElement('div');
-    button.classList.add('btn-copy');
-    button.title = 'copy to clipboard';
-    button.addEventListener('click', () => {
-        try {
-            copyText(elem.innerText);
-            button.title = 'copied!'
-            button.classList.add('btn-copy-ok')
-            setTimeout(() => {
-                button.classList.remove('btn-copy-ok')
-            }, 3000);
-        } catch (e) {
-            console.error(e);
-            button.classList.add('btn-copy-error')
-            setTimeout(() => {
-                button.classList.remove('btn-copy-error')
-            }, 3000);
-        }
-    });
-    elem.prepend(button);
-}
+window.onCopyClick = (button) => {
```

Comment:
We would never to a global function in PHP. 

Im not sure what is best here. Either we are happy with the global function, or we register a bunch of event listeners (which takes time) and that we may never need..

Hm.. Im fine with a global function.

Reply:
you're right. I'm fixing it